gulpfile.js

```diff
@@ -69,7 +69,7 @@ gulp.task('fixup-swagger', function () {
   autorest.fixupRawSwagger('./swagger/bifrost.swagger.before.json', './swagger/bifrost.swagger.json');
 });
 
-gulp.task('autorest', ['clean-autorest', 'fixup-swagger'], function () {
```

Comment:
Without the clean-autorest target in here, we could get leftover files in the client code if for example operations were removed from the swagger. Why did you take this out?